manifest_staging/charts/secrets-store-csi-driver/templates/_helpers.tpl

```diff
@@ -35,6 +35,9 @@ labels:
   app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
   app: {{ template "sscd.name" . }}
   helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+  {{- if .Values.commonLabels}}
+  {{ toYaml .Values.commonLabels }}
+  {{- end }}
```

Comment:
@thib-mary fyi `sscd.labels` is not added in all templates.

Reply:
> @thib-mary fyi `sscd.labels` is not added in all templates.

Indeed, can I still update? @nilekhc

Reply:
Sure. I noted this to make sure it satisfies your use case.